public/js/src/module/file-manager.js

```diff
@@ -72,7 +72,14 @@ function getFileUrl( subject ) {
             if ( isTooLarge( subject ) ) {
                 reject( _getMaxSizeError() );
             } else {
-                resolve( URL.createObjectURL( subject ) );
+                const objectURL = URL.createObjectURL( subject );
+                urlToBlob( objectURL )
+                    .then( resizedBlob => {
+                        resizedFileBlob = resizedBlob;
```

Comment:
Based on enketo-core commit, `getFileUrl` will be called when filePicker resize the images(`_resizeFile`) then send the blobs of resized images to `getFileUrl` so, in the updated code, I still store the blobs of resized images directly from the `subject` param, not from `urlToBlob`. I think instead of using `urlToBlob` in `getCurrentFiles` to get the blob of resized images, it's faster to use the blob that already created by filePicker from `getFileUrl`. 
Please let me know if it's still unacceptable.